File: taoyoupin-manager-web/src/main/webapp/service/brandService.js
app.service('brandService' ,function($http){
    //读取列表数据绑定到表单中
    this.findAll=function(){
        return $http.get('../brand/findAll.do');
    };


    this.reLoadList=function(currentPage,itemsPerPage,searchEntity){
        return this.search(currentPage,itemsPerPage,searchEntity);
    };



    this.findPage=function(page,rows) {
        return $http.get('../brand/findPage.do?page=' + page + '&rows=' + rows);
    };


    this.search=function(page,rows,searchEntity) {
        return $http.post('../brand/search.do?page=' + page + '&rows=' + rows,/*$scope.*/searchEntity);
    };

    this.save=function(methodName,entity) {


        return $http.post('../brand/'+methodName+'.do?',entity);
    };


    this.findOne=function(id){
        return $http.get('../brand/findOne.do?id='+id);
    };




    this.dele=function(selectIds){
        //获取选中的复选框
        return $http.get('../brand/delete.do?ids='+/*$scope.*/selectIds);
    };

});

File: taoyoupin-sellergoods-interface/src/main/java/com/taoyoupin/sellergoods/service/BrandService.java
package com.taoyoupin.sellergoods.service;



import com.taoyoupin.entity.PageResutl;
import com.taoyoupin.pojo.TbBrand;

import java.util.List;

public interface BrandService {

    public List<TbBrand> findAll();

    PageResutl findPage(Integer curPage,Integer pageSize);

    void add(TbBrand tbBrand);

    TbBrand findOne(Long id);

    void update(TbBrand tbBrand);

    void delete(Long[] ids);

    PageResutl search(TbBrand tbBrand,Integer curPage,Integer pageSize);

}
